src/App.js

```diff
@@ -1,9 +1,48 @@
-import './App.css';
+import "./App.css";
+import React, { useState } from "react";
+import BoardList from "./components/BoardList";
+
+function App() {
+  const [allBoardData, setAllBoardData] = useState([]);
+  const [selectedBoard, setSelectedBoard] = useState(null);
+
+  // Simulation to test
+  const sampleBoardData = [
+    { id: 1, title: "Portfolio examples", owner: "Barbara" },
+    { id: 2, title: "Leetcode for beginners", owner: "Elizabeth" },
+    { id: 3, title: "Rock your interview", owner: "Josh" },
+  ];
+
+  const handleBoardClick = (title, owner, id) => {
+    setSelectedBoard({ title, owner, id });
+  };
 
-const App = () => {
   return (
-    <div className="App">
-      <h1>Hello World</h1>
+    <div>
+      <h1> Inspiration Board </h1>
+      <section className="board__container">
```

Comment:
I might have missed this in class, but is the double underscore for classnames a React convention?